templates/common/points_indicator/component.html

```diff
@@ -0,0 +1,30 @@
+<div class="points-indicator--{{ result }}">
+	<svg class="icon">
```

Comment:
Should this be `points-indicator__icon`?

Reply:
I will swap it that way to avoid any risks, yes.

The idea here was that all styles are nested inside the container, so shouldn't leak out, but I guess things leaking "in" might be as much of a concern.

Reply:
> I guess things leaking "in" might be as much of a concern.

I have noticed that to be challenging. If everything would be perfect, it would be nice :slightly_smiling_face: